README.md

```diff
@@ -1,23 +1,98 @@
 [![Build Status](https://travis-ci.org/levigo/url-crypto-kit.svg?branch=master)](https://travis-ci.org/levigo/url-crypto-kit)
 [![Generic badge](https://img.shields.io/badge/current%20version-1.3.3-1abc9c.svg)](https://github.com/levigo/url-crypto-kit/tree/v1.3.3)
 
-# URL cryptography functionality to be used in conjunction with the Spring(tm) Framework 
+# URL cryptography functionality to be used in conjunction with the Spring&trade; Framework 
 
 ## Features
 - Pre-signed-URL generation and verification
 - URL encryption and decryption
 - integration with the Spring Framework
 - integration with Spring Security
-- Activation and configuration through Spring-Boot auto configuration
+- Activation and configuration through Spring-Boot autoconfiguration
+
+## PSU
+P(re) S(igned) U(RL) is a mechanism which enables an authenticated user to create a URL with a built-in authentication 
+for a single endpoint.  
+The PSU will enable anyone to use the requested endpoint, this URL is created for, without any additional 
+authentication.   
+This can be useful for services that can not perform the necessary authentication or simply don't have the required 
+access rights.  
+A user can only create a PSU for an endpoint he is authenticated for.  
+The Access rights of a PSU are equivalent to the user who have created it.  
+PSU are always time limited and will expire after a defined time. If the link is expired it will not provide any 
+authentication.
 
 ## Usage
-__Maven dependency__
+### Maven dependency
+```xml
+<dependency>
+    <groupId>com.neverpile</groupId>
+    <artifactId>url-crypto-kit</artifactId>
+    <version>1.3.3</version>
+</dependency>
+```
+### Example configuration
+application.properties:  
+```properties
+neverpile.url-crypto.shared-secret.enabled=true
+neverpile.url-crypto.shared-secret.secret-key=Not#So%Secret
+neverpile.url-crypto.pathPatterns=/**
+neverpile.url-crypto.enabledStaticPaths=/path/to/static/data/**
+```
+__neverpile.url-crypto.shared-secret.enabled=true__  
+Enable the CryptoKit with a shard secret implementation.  
+This is the only implementation as of now.  
+__neverpile.url-crypto.shared-secret.secret-key=Not#So%Secret__  
+Choose a secret key for the encryption.  
+Please don't copy the example key.  
+__neverpile.url-crypto.pathPatterns=/**__  
+Allows you to define one or multiple URL patterns where the PSU mechanism should be activated on.  
+All URL paths which do not match any of the defined paths will not be considered for encryption.  
+/&ast;&ast; will allow PSU to be enabled on all paths.  
+__neverpile.url-crypto.enabledStaticPaths=/path/to/static/data/**__  
+All paths defined here should also match a pattern in the global pathPatterns property.  
+Paths defined here allow the use of PSU without any additional configuration.
```

Comment:
Ich muss gestehen, ich versteh nicht, was mit "without any additional configuration" gemeint ist. Kannst du genauer erläutern, wie man das verwendet?

Evtl. die Property aus Konsistenzgründen umbenennen in "staticPathPatterns", dann passt die Bezeichnung besser zu "pathPatterns"?

Reply:
das ding heißt schon `psuEnabledPathPatterns` ich hab's nur in der readme falsch gehabt.